zebra/rt_netlink.c

```diff
@@ -1891,7 +1891,7 @@ static int netlink_vxlan_flood_list_update(struct interface *ifp,
 		char buf[256];
 	} req;
 	uint8_t dst_mac[6] = {0x0, 0x0, 0x0, 0x0, 0x0, 0x0};
-	struct zebra_vrf *zvrf = zebra_vrf_lookup_by_id(ifp->vrf_id);
+	struct zebra_vrf *zvrf = (struct zebra_vrf *)vrf_to_vrf_info(ifp->vrf);
```

Comment:
If we are removing the zebra_vrf_lookup_by_id then we don't really need to have the (struct zebra_vrf *) cast as that void * pointers will be auto cast here.

Reply:
ok